testsprite_tests/TC002_User_Authentication_with_Invalid_Credentials.py

```
@@ -0,0 +1,93 @@
+        frame = context.pages[-1]
+        # Click the 'INITIATE SESSION' button to submit the login form with invalid credentials
+        elem = frame.locator('xpath=html/body/div[2]/div[2]/div[3]/form/div[2]/button').nth(0)
+        await page.wait_for_timeout(3000); await elem.click(timeout=5000)
+        
+
+        # --> Assertions to verify final state
+        frame = context.pages[-1]
+        try:
+            await expect(frame.locator('text=Login Successful').first).to_be_visible(timeout=3000)
+        except AssertionError:
+            raise AssertionError("Test failed: Login should be rejected with invalid credentials, but 'Login Successful' message was not expected.")
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)